lua/cl_discord.lua

```diff
@@ -1,6 +1,6 @@
 // This requires a special module to be installed before it works correctly
 // Sorry to disappoint you
-if not file.Exists('lua/bin/gmcl_fluffydiscord_win32.dll', 'GAME') and not file.Exists('lua/bin/gmcl_fluffydiscord_win64.dll', 'GAME') and not file.Exists('lua/bin/gmcl_fluffydiscord_linux.dll', 'GAME') then return end
+if not file.Exists('lua/bin/gmcl_fluffydiscord_win32.dll', 'GAME') and not file.Exists('lua/bin/gmcl_fluffydiscord_win64.dll', 'GAME') and not file.Exists('lua/bin/gmcl_fluffydiscord_linux.dll', 'GAME') and not file.Exists('lua/bin/gmcl_fluffydiscord_linux64.dll', 'GAME') then return end
```

Comment:
This line is getting a bit too long, would you be able to clean this up to something a bit nicer?

Reply:
Should be better now.